src/function/vector_hash_functions.cpp

```diff
@@ -13,46 +13,73 @@ void VectorHashFunction::computeHash(ValueVector* operand, ValueVector* result)
     switch (operand->dataType.getPhysicalType()) {
     case PhysicalTypeID::INTERNAL_ID: {
         UnaryHashFunctionExecutor::execute<internalID_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::BOOL: {
         UnaryHashFunctionExecutor::execute<bool, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::INT64: {
         UnaryHashFunctionExecutor::execute<int64_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::INT32: {
         UnaryHashFunctionExecutor::execute<int32_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::INT16: {
         UnaryHashFunctionExecutor::execute<int16_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::INT8: {
         UnaryHashFunctionExecutor::execute<int8_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::UINT64: {
         UnaryHashFunctionExecutor::execute<uint64_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::UINT32: {
         UnaryHashFunctionExecutor::execute<uint32_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::UINT16: {
         UnaryHashFunctionExecutor::execute<uint16_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::UINT8: {
         UnaryHashFunctionExecutor::execute<uint8_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::DOUBLE: {
         UnaryHashFunctionExecutor::execute<double, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::FLOAT: {
         UnaryHashFunctionExecutor::execute<float_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::STRING: {
         UnaryHashFunctionExecutor::execute<ku_string_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
     case PhysicalTypeID::INTERVAL: {
         UnaryHashFunctionExecutor::execute<interval_t, hash_t>(*operand, *result);
-    } break;
+    }
+        return;
+    case PhysicalTypeID::STRUCT: {
+        if (operand->dataType.getLogicalTypeID() == LogicalTypeID::NODE) {
+            assert(0 == common::StructType::getFieldIdx(&operand->dataType, InternalKeyword::ID));
+            auto idVector = StructVector::getFieldVector(operand, 0);
+            UnaryHashFunctionExecutor::execute<internalID_t, hash_t>(*idVector, *result);
+            return;
+        } else if (operand->dataType.getLogicalTypeID() == LogicalTypeID::REL) {
+            assert(3 == StructType::getFieldIdx(&operand->dataType, InternalKeyword::ID));
+            auto idVector = StructVector::getFieldVector(operand, 3);
+            UnaryHashFunctionExecutor::execute<internalID_t, hash_t>(*idVector, *result);
```

Comment:
Do we have a test that computes the hash for node/rel? Not sure why it is not covered by test